gitpod-io/gitpod
# supervisor: Remove slirp4netns

## Description


This PR is a follow-up PR to #9212 
no3 of https://github.com/gitpod-io/gitpod/issues/8106#issuecomment-1094418016

## Related Issue(s)

Relates https://github.com/gitpod-io/gitpod/issues/8106

## How to test


on gitpod
https://to-veths-again3.staging.gitpod-dev.com/workspaces
```console
$ docker run hello-world
$ python -m http.server 5000 # and ensure to be able to access 5000 port from outward
```

## Release Notes

```release-note
supervisor: Remove slirp4netns
```

## Documentation
